javascript_client/src/subscriptions/__tests__/createActionCableHandlerTest.ts

```diff
@@ -51,14 +51,16 @@ describe("createActionCableHandler", () => {
       {text: "", name: "", id: "abcdef"},
       {},
       {},
-      { onError: () => {}, onNext: () => {}, onCompleted: () => {} }
+      { onError: () => {}, onNext: (result: any) => { log.push(["onNext", result])}, onCompleted: () => { log.push(["onCompleted", null])} }
     )
 
     handlers.connected() // trigger the GraphQL send
+    handlers.received({ result: { data: { a: "1" } }, more: false })
 
     expect(log).toEqual([
-      ["send", { operationId: "client-1/abcdef", operationName: "", query: "", variables: {} }],
       ["execute", { operationId: "client-1/abcdef", operationName: "", query: "", variables: {} }],
+      ["onNext", { data: { a: "1" } }],
+      ["onCompleted", null],
```

Comment:
I assume that the previous assertion here was added after-the-fact, based on the assumption that the code worked. I'm removing that -- and adding tests for a couple other hooks in the meantime.